src/Classes/Factories/AddEventPageControllerFactory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Portal\Factories;
+
+use Portal\Controllers\AddEventPageController;
+
+class AddEventPageControllerFactory
+{
+    /**
+     * @param $c
+     * @return AddEventPageController
+     */
+    public function __invoke($c)
```

Comment:
[birds/plane/etc comment from previous review]

Reply:
Renamed in 114d7f3